### PR TITLE
CMB/MP gear loadout edits, Mod 88 edits, CMB/MP resprites

### PR DESCRIPTION
**Pull request by AmoryBlaine**
- Updates CMB jackets to reflect peerage with Hybrisa PD jackets.
- Changes wood palette of highpower pistols
- Replaces emergency syringe kit from Deputy ERT with alt kit (bandages, gauze, tri, splints)
- Gets rid of beanbag handful in favour of more slugs
- Replaces boots on Deputy and Marshal with laceup shoes
- Replaces MP5 Deputy with rarer M41A Deputy
- Gives all CMBs flare boxes
- Gets rid of extra misc items unimportant to CMB RP and gameplay (cigarette, binocs, legacy holobadge, holdout pistol, sechud, ect) This gives you room to pick things up or spawn in with your loadout gear
- Changes wording of CMB jacket desc
- Makes the 88 Mod use FMJ rounds, also makes it deal no additional damage that was intended to work in tandem with the AP rounds it fired
- Makes the MP uniform into a standard BDU outfit with a red Hybrisa PD armband

USCM MP, CMB Deputy, CMB Marshal
image

### CODE REVIEW

**Comment by AmoryBlaine** [comment]
Stopping myself here cause I will keep adding shit like a mad man otherwise

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

File: code/game/machinery/vending/vending_types.dm [hunk 1]
```diff
@@ -222,36 +222,45 @@
...
-		/obj/item/clothing/glasses/sunglasses/sechud = 3,
-		/obj/item/device/radio/headset = 6,
 		/obj/item/tape/regulation = 5,
 	)
 	contraband = list(/obj/item/clothing/glasses/sunglasses = 2,/obj/item/storage/donut_box = 2)
 
+/obj/structure/machinery/vending/security/colony
+	name = "\improper SecTech Police Vendor"
+	desc = "A generic police vendor."
+	product_ads = "Crack capitalist skulls!;Beat some heads in!;Don't forget - harm is good!;Your weapons are right here.;Handcuffs!;Freeze, scumbag!;Don't tase me bro!;Tase them, bro.;Why not have a donut?"
```
[the remark this answers is not in the document]
Comment (AmoryBlaine) [reply code/game/machinery/vending/vending_types.dm:250]: Done

**Review by Doubleumc — APPROVED** [review]
Code looks good to me.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Pull request merged**